docs/1.5/extensions/external-links.md

```diff
@@ -30,9 +31,12 @@ $environment->addExtension(new ExternalLinkExtension());
 // Set your configuration
 $config = [
     'external_link' => [
-        'internal_hosts' => 'www.example.com', // Don't forget to set this!
+        'internal_hosts' => 'www.example.com', // TODO: Don't forget to set this!
         'open_in_new_window' => true,
         'html_class' => 'external-link',
+        'nofollow' => 'external',
```

Comment:
This should default to empty as it has serious SEO consequences and should be left as opt-in.

Reply:
These are just examples of possible values but I'll change them to be more-sane for anyone blindly copypasting.  Although I did actually have `nofollow` set to `external` by default, which I've changed via commit 3477aa21